are_we/tools/wav_to_array.py

```diff
@@ -0,0 +1,49 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io.wavfile
+import time
+import math
+#from matplotlib import animation
+
+
+#rate, audio_data = scipy.io.wavfile.read("/home/pi/Development/view-house-lights/are_we/tools/Lube_-_Kombat-alejKZhngvg.wav")
+def parse():
+    rate, audio_data = scipy.io.wavfile.read("/home/pi/Development/view-house-lights/are_we/tools/lowered.wav")
+    tt = audio_data.shape[0] / rate
+    print("Total time of piece is: {}secs".format(tt))
+
+    # grab a smaller portion of our audio
+    amount = 60*rate
+    audio_data = audio_data[(120*rate):(120*rate)+amount]
+
+    # int16 -> int32. Else overflow
+    audio_data = audio_data.astype(np.int32)
+
+    mn = np.min(audio_data)
+    mx = np.max(audio_data)
+    
+    # Stopping here gives a direct conversion of the -32xxx:32xxx
+    # range onto the range 0:255
+    output = ((audio_data-mn)*255.0) / (mx-mn)
+
+
+    amplitude = [(x-127)/255.0 for x in output]
```

Comment:
Should this take advantage of numpy vectorization instead of pure python list comprehensions?

Reply:
Yes, in general. Two things though, 
1) The speed that the arduino can read from serial without getting confused is much much slower than the slowdown this inefficient code make
2) This ideally will be abstracted away by a dedicated library (pyaudio) in the future once we get the ALSA loopback record working correctly (I'd put this effort at 50% complete)

Think of this as a super naive implementation

Reply:
Alright, I would put a todo so this doesn't become a "temporary solution".